ruma-federation-api/src/discovery/query_keys/v2.rs

```diff
@@ -0,0 +1,38 @@
+//! [GET /_matrix/key/v2/query/{serverName}/{keyId}](https://matrix.org/docs/spec/server_server/r0.1.4#get-matrix-key-v2-query-servername-keyid)
+
+use std::time::SystemTime;
+
+use crate::discovery::ServerKey;
+use ruma_api::ruma_api;
+
+ruma_api! {
+    metadata {
+        description: "Query for another server's keys.",
+        method: GET,
+        name: "query_keys",
```

Comment:
We normally use the verbs `get`, `create`, `send` and `update` for the request names. I think something like `get_remote_server_keys` (and `get_remote_server_keys_batch` for the corresponding batch endpoint) could work. That would also help to make the distinction between `get_server_keys`, which gets the keys for the homeserver that the request is sent to, and this one, which sends requests queries of other servers through a notary.

What do you think?

Reply:
Yep, that makes perfect sense, adjusted. Thanks for the explanation, and sorry that I haven't paid enough attention to the existing conventions.